Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/SeedingAlgorithm.hpp

```diff
@@ -64,6 +64,10 @@ class SeedingAlgorithm final : public IAlgorithm {
     // number of phiBin neighbors at each side of the current bin that will be
     // used to search for SPs
     int numPhiNeighbors = 1;
+
+    // Connect custom selections on the space points or to the doublet
+    // compatibility
+    bool useExperimentCuts = false;
```

Comment:
It doesn't _really_ matter, but in the Examples the cuts are sort of fixed. OTOH calling it "Experiment" cuts illustrates what this can be good for.

Reply:
Hi @paulgessinger, 
I didn't like too much to have fixed cuts, but I also thought that the signature of one of the function for the experimentCuts is quite restrictive (only accepts 2 floats). We could change it to get a space point instead, but would break Athena code as well.

The code in Examples is for providing examples after all =) 
Do you have suggestions or this sounds reasonable to you?

Reply:
Changed name to `useExtraCuts`